automation_tools/__init__.py

```diff
@@ -1122,6 +1122,13 @@ def product_install(distribution, create_vm=False, certificate_url=None,
                 setup_proxy, host=host, run_katello_installer=False
             )[host])
 
+    if os.environ.get('INSTALLER_OPTIONS'):
```

Comment:
What about:

```python
    if os.environ.get('INSTALLER_OPTIONS'):
        # INSTALLER_OPTIONS are comma separated katello-installer options.
        # It will be of the form "key1=val1,key2=val2".
        installer_options.update(dict([
            opt.split('=') for opt in os.environ.get('INSTALLER_OPTIONS').split(',')
        ]))
```